forge-gui/res/cardsfolder/upcoming/their_number_is_legion.txt

```diff
@@ -0,0 +1,12 @@
+Name:Their Number Is Legion
+ManaCost:X B B B B
+Types:Sorcery
+A:SP$ Token | TokenAmount$ X | TokenScript$ b_2_2_a_necron_warrior | SubAbility$ DBGainLife | SpellDescription$ Create X tapped 2/2 black Necron Warrior artifact creature tokens, then you gain life equal to the number of artifacts you control. Exile CARDNAME.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ Y | StackDescription$ SpellDescription | SubAbility$ DBChange
+SVar:DBChange:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | SpellDescription$ Exile CARDNAME.
```

Comment:
Won't the Exile SpellDescription be displayed twice now?

Reply:
Still happens:
![image](https://user-images.githubusercontent.com/8506892/191836854-c8ac1cf1-f380-4004-b5ca-044b6c8cd0e2.png)